ext/iconv/tests/iconv_mime_encode_001.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+iconv_mime_encode(): Encoding parameter exceeds the maximum allowed for preference "input-charset"
+--CREDITS--
+Juan Carlos Morales (jcmargentina@gmail.com)
+--EXTENSIONS--
+iconv
+--INI--
+iconv.internal_charset=iso-8859-1
+--FILE--
+<?php
+$extra = str_repeat("x", 256);
+
+$preferences = array(
+    "input-charset" => "ISO-8859-1".$extra,
+    "output-charset" => "UTF-8",
+    "line-length" => 76,
+    "line-break-chars" => "\n"
+);
+$preferences["scheme"] = "B";
+
+$result = iconv_mime_encode("Subject", "Prüfung", $preferences);
+var_dump($result);
+?>
+--EXPECTF--
+%s Encoding parameter exceeds the maximum allowed length of %s
```

Comment:
It may make sense to expect "Warning: iconv_mime_encode(): " at the start of the line, not some arbitrary string. I also think it's better to match more of the error message explicitly, i.e.
```suggestion
Warning: iconv_mime_encode(): Encoding parameter exceeds the maximum allowed length of %d characters in %s on line %d
```